versions.json

```diff
@@ -1,6 +1,6 @@
 {
     "react": "9.0.0",
-    "main": "9.10.0",
+    "main": "9.10.1",

```

Comment:
Hmm, I think we only need to bump the plugins package. And not all the other packages.

Reply:
Good point!